O365WebHook/o365content.js

```diff
@@ -25,44 +25,51 @@ const g_ingestc = new m_ingest.Ingest(
         }
 );
 
+// One O365 content message is about 1KB.
+var MAX_BATCH_MESSAGES = 1500;
```

Comment:
I would rather decrease it. Do we have 2 MB limit or 1 MB ? If the latter one I would suggest to keep it close to 1000.

Reply:
We currently have about 1MB payload limit of Ingest input. 1500 should be ok as the compression is always applied to a batch before transfer.